runtime/compiler/env/VMJ9.cpp

```diff
@@ -6715,6 +6715,12 @@ TR_J9VM::classHasBeenReplaced(TR_OpaqueClassBlock * clazzPointer)
 
 bool
 TR_J9VMBase::isGetImplInliningSupported()
```

Comment:
After delivering this PR, maybe we can change OMRCompilation.cpp to not use
`self()->setGetImplInlineable(self()->fej9()->isGetImplInliningSupported());` anymore, so that we can eliminate `TR_J9VMBase::isGetImplInliningSupported()` and rely only on `TR_J9VMBase::isGetImplAndRefersToInliningSupported()`

Reply:
Thanks, Marius @mpirvu.  Yes - I have another pair of pull requests that I'll open to do that.  One to change OMRCompilation.cpp, and another to remove the `getImpl`-specific methods from OpenJ9.